lib/Github/Api/Repository/Checks/CheckRuns.php

```diff
@@ -83,4 +83,16 @@ public function allForReference(string $username, string $repository, string $re
 
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/commits/'.rawurlencode($ref).'/check-runs', $params);
     }
+
+    /**
+     * @link https://docs.github.com/en/rest/reference/checks#rerequest-a-check-run
+     *
+     * @return array
+     */
+    public function rerequest(string $username, string $repository, int $checkRunId)
+    {
+        $this->acceptHeaderValue = 'application/vnd.github+json ';
```

Comment:
@Spea this is the default header so it doesn't need to be set here. The client will apply that one by default. So if you can remove this line, the PR is good to go!

https://github.com/KnpLabs/php-github-api/blob/845545b0686e0dbbafe0972a8b5b9f86be856976/lib/Github/Client.php#L136-L139

Reply:
Thank you for pointing that out. I removed it now.